prime-router/settings/prod/0098-update-nv-phd-data-quality.yml

```diff
@@ -0,0 +1,105 @@
+# ./prime multiple-settings set --env prod --input ./settings/prod/0098-update-nv-phd-data-quality.yml
+---
+- name: "nv-phd"
+  description: "Nevada Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "NV"
+  countyName: null
+  filters:
+    - topic: "covid-19"
+      jurisdictionalFilter:
+        - "orEquals(ordering_facility_state, NV, patient_state, NV)"
+      qualityFilter:
+        - "hasValidDataFor(message_id,equipment_model_name,specimen_type,test_result,patient_last_name,patient_first_name,patient_dob)"
+        - "hasAtLeastOneOf(patient_street,patient_zip_code,patient_phone_number,patient_email)"
+        - "hasAtLeastOneOf(order_test_date,specimen_collection_date_time,test_result_date)"
+        - "isValidCLIA(testing_lab_clia,reporting_facility_clia)"
+        - "doesNotMatch(processing_mode_code,T,D)"
+      routingFilter: null
+      processingModeFilter: null
+  senders: []
+  receivers:
+    - name: "elr"
+      organizationName: "nv-phd"
+      topic: "covid-19"
+      customerStatus: "testing"
+      translation: !<HL7>
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "NVDOH"
+        receivingApplicationOID: null
+        receivingFacilityName: "NVODH"
+        receivingFacilityOID: null
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB:
+          ORC-12-2: ["":"Unknow", "Crona":"Joe","Bode":"John;@:,"]
```

Comment:
@oslynn `unknown` is misspelled. Please fix. What are the other values for? Are those test values?